proposals/0387-bls-pubkey-management-in-vote-account.md

```diff
@@ -0,0 +1,222 @@
+---
+simd: '0387'
+title: BLS Pubkey management in vote account
+authors:
+  - Sam Kim (Anza)
+  - Quentin Kniep (Anza)
+  - Wen Xu (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2025-10-27
+feature: (fill in with feature key and github tracking issues once accepted)
+---
+
+## Summary
+
+This proposal specifies in detail how a BLS public key can be generated by
+users via updated existing tools and how they can put the generated BLS public
+keys into their vote accounts for voting in Alpenglow.
```

Comment:
We should mention a critical piece of the proposal here, which is how the network verifies PoP